TestCases/deformation/intersection_prevention/def_intersect.cfg

```diff
@@ -0,0 +1,452 @@
+
+% Output file convergence history (w/o extension) 
+CONV_FILENAME= history
+
+% Output file restart flow
+%RESTART_FLOW_FILENAME= restart_flow.dat
+RESTART_FILENAME= restart_flow.dat
+
+% Output file restart adjoint
+RESTART_ADJ_FILENAME= restart_adj.dat
+
+% Output file flow (w/o extension) variables
+%VOLUME_FLOW_FILENAME= flow
+VOLUME_FILENAME= flow
+
+% Output file adjoint (w/o extension) variables
+VOLUME_ADJ_FILENAME= adjoint
+
+% Output objective function gradient (using continuous adjoint)
+GRAD_OBJFUNC_FILENAME= of_grad.dat
+
+% Output file surface flow coefficient (w/o extension)
+%SURFACE_FLOW_FILENAME= surface_flow
+SURFACE_FILENAME= surface_flow
+
+% Output file surface adjoint coefficient (w/o extension)
+SURFACE_ADJ_FILENAME= surface_adjoint
+
+VOLUME_OUTPUT= RESIDUAL PRIMITIVE SOURCE SENSITIVITY COEFFICIENT
+
+% Writing convergence history frequency
+SCREEN_WRT_FREQ_INNER = 1
+SCREEN_WRT_FREQ_OUTER = 1
+
+% ----------------------- DESIGN VARIABLE PARAMETERS --------------------------%
+%
+% Kind of deformation (NO_DEFORMATION, TRANSLATION, ROTATION, SCALE,
+%                      FFD_SETTING, FFD_NACELLE
+%                      FFD_CONTROL_POINT, FFD_CAMBER, FFD_THICKNESS, FFD_TWIST
+%                      FFD_CONTROL_POINT_2D, FFD_CAMBER_2D, FFD_THICKNESS_2D, FFD_TWIST_2D,
+%                      HICKS_HENNE, SURFACE_BUMP)
+
+DV_KIND= FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D, FFD_CONTROL_POINT_2D 
+
+MESH_FILENAME = mesh_ffd.su2
+
+% Mesh output file
+MESH_OUT_FILENAME= mesh_ffd_deformed.su2
+
+% Marker of the surface in which we are going apply the shape deformation
+DV_MARKER= (wall_inner_bottom, wall_inner_right, wall_inner_top)
+
+% Parameters of the shape deformation
+DV_PARAM= (MAIN_BOX, 0, 0, 0.0, 1.0); (MAIN_BOX, 1, 0, 0.0, 1.0); (MAIN_BOX, 2, 0, 0.0, 1.0); (MAIN_BOX, 3, 0, 0.0, 1.0); (MAIN_BOX, 4, 0, 0.0, 1.0); (MAIN_BOX, 5, 0, 0.0, 1.0); (MAIN_BOX, 6, 0, 0.0, 1.0); (MAIN_BOX, 7, 0, 0.0, 1.0); (MAIN_BOX, 8, 0, 0.0, 1.0); (MAIN_BOX, 9, 0, 0.0, 1.0); (MAIN_BOX, 10, 0, 0.0, 1.0); (MAIN_BOX, 0, 1, 0.0, 1.0); (MAIN_BOX, 1, 1, 0.0, 1.0); (MAIN_BOX, 2, 1, 0.0, 1.0); (MAIN_BOX, 3, 1, 0.0, 1.0); (MAIN_BOX, 4, 1, 0.0, 1.0); (MAIN_BOX, 5, 1, 0.0, 1.0); (MAIN_BOX, 6, 1, 0.0, 1.0); (MAIN_BOX, 7, 1, 0.0, 1.0); (MAIN_BOX, 8, 1, 0.0, 1.0); (MAIN_BOX, 9, 1, 0.0, 1.0); (MAIN_BOX, 10, 1, 0.0, 1.0) 
+
+% Value of the shape deformation
+DV_VALUE = 0, 0.0002, 0.0004, 0.0006, 0.0008, 0.001, 0.0012, 0.0014, 0.0016, 0.0018, 0.002, 0,-0.0002, -0.0004, -0.0006, -0.0008, -0.0010, -0.0012, -0.0014, -0.0016, -0.0018, -0.002

```

Comment:
Is the prevention active for these values? If it is LGTM

Reply:
yes, this is the case from the thesis that gave problems. Thanks!